cloudify_agent/api/pm/detach.py

```diff
@@ -64,11 +64,62 @@ def before_self_stop(self):
         self._runner.run(self.create_disable_cron_script())
         super(DetachedDaemon, self).before_self_stop()
 
+    def _clean_queue(self, client):
+        queue = '{0}_service'.format(self.queue)
+        self._logger.debug('Cleaning amqp agent queue {0}'.format(queue))
```

Comment:
```suggestion
        self._logger.debug('Cleaning amqp agent queue %s', queue)
```
as always, the usual comment about str.format and logging
(no need to format the string if it's not going to be emitted, like in case of debug logs)